src/rust/engine/client/src/main.rs

```diff
@@ -122,6 +122,8 @@ const PANTS_NO_NATIVE_CLIENT: &str = "PANTS_NO_NATIVE_CLIENT";
 
 #[tokio::main]
 async fn main() {
+    grpc_util::initialize().expect("grpc_util init");
```

Comment:
Initializes TLS algorithms in-process. Needs to be called as early as possible.

Reply:
I don't know currently since I didn't dig in on the initialization flow. Definitely a place where we might benefit from documenting the various startup flows or consolidating them.